cmd/operator/run.go

```diff
@@ -55,15 +56,20 @@ func kill(stopCh chan struct{}) {
 	os.Exit(0)
 }
 
-// runProtoform ...
+// runProtoform will add CRD controllers to the Protoform Deployer which
+// will call each of their Deploy functions
 func runProtoform(configPath string) {
+	// Add controllers to the Protoform Deployer
 	deployer, err := protoform.NewController(configPath)
 	if err != nil {
 		panic(err)
 	}
 
 	stopCh := make(chan struct{})
 
+	sampleController := sample.NewCRDInstaller(deployer.Config, deployer.KubeConfig, deployer.KubeClientSet, bdutil.GetSampleDefaultValue(), stopCh)
```

Comment:
This should be commented out. I don't think that we want to deploy the sample CRD with every installation of the synopsys operator.

Reply:
I commented this out. I also commented out the GetSampleDefaultValue() function in apps/util/crd_defaults.go